packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -3,26 +3,24 @@
 // Omitting rgb "fire" style, so no `chalk` dependency
 // Omitting: NaN handling, special consideration for constant data.
 
-let create = (
-  numbers: array<float>,
-  ~minimum=?,
-  ~maximum=?,
-  ()
-) => {
-  let ticks = [`▁`, `▂`, `▃`, `▄`, `▅`, `▆`, `▇`, `█`]
+let ticks = [`▁`, `▂`, `▃`, `▄`, `▅`, `▆`, `▇`, `█`]
 
-  let minimum = E.O.default(Js.Math.minMany_float(numbers), minimum)
-  let maximum = E.O.default(Js.Math.maxMany_float(numbers), maximum)
+let _ticksLength = E.A.length(ticks)
 
-  let toHeight = (number: float) => {
-    let tickIndex = Js.Math.ceil_int((number /. maximum) *. (ticks -> Belt.Array.length -> Belt.Int.toFloat)) - 1
-    let tickIndex = if maximum == 0.0 || tickIndex < 0 {
-        0
-      } else {
-        tickIndex
-      }
-    ticks[tickIndex]
-  }
+let _heightToTickIndex = (maximum: float, v: float) => {
+  let v = Js.Math.ceil_int(v /. maximum *. Belt.Int.toFloat(_ticksLength)) - 1
+  min(v, 0)
+}
 
-  toHeight -> E.A.fmap(numbers) -> (arr => E.A.joinWith("", arr))
+let create = (relativeHeights: array<float>, ~maximum=?, ()) => {
+  if E.A.length(relativeHeights) === 0 {
+    ""
+  } else {
+    let maximum = maximum |> E.O.default(E.A.max(relativeHeights) |> E.O.toExn(""))
+
+    relativeHeights
+    |> E.A.fmap(_heightToTickIndex(maximum))
```

Comment:
I'm not sure we in fact want to dig ourselves further into the deprecated `|>`? Ways I can imagine myself not worrying about it is 
1. we commit to freezing rescript version increase if there's ever a release that no longer supports it
2. we assume that we won't refactor them all into `->` until, like, a couple years down the road (if we're still in rescript by then)